CMakeLists.txt

```diff
@@ -94,3 +98,37 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
+    # General settings
+    set(CPACK_GENERATOR "TBZ2")
+    set(CPACK_PACKAGE_NAME "libqhotkey")
+    set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Global hotkey library for Qt software")
+    set(LICENSE_FILE "LICENSE")
+    set(README_FILE "README.md")
+    set(PROJECT_URL "https://skycoder42.github.io/QHotkey/")
+    set(COMPRESION_TYPE "xz")
+    set(CPACK_PACKAGE_VENDOR "Javier O. Cordero Pérez")
+    set(CPACK_PACKAGE_CONTACT "javiercorderoperez@gmail.com")
```

Comment:
Who is this?

Reply:
That is me, as mentioned in the last paragraph of my initial comment:
> I've set myself as package vendor and contact, but anyone's free to take the reigns (of maintaining the package). ...

Would you prefer me to write your contact info or should we write Skycoder42's? You are undoubtedly better versed in CMake than I am, and are already involved with code maintenance, thus it might be best if you maintained the package.

Reply:
Oh, got it, sorry.
Yes, I think we should write Skycoder42. But it looks like he does not want to post email.

Reply:
Either `CPACK_PACKAGE_CONTACT` or `CPACK_DEBIAN_PACKAGE_MAINTAINER` must be specified. People usually use an e-mail, or a name and an e-mail, but contact could also be just a name. I suggest we write:
```
     set(CPACK_PACKAGE_VENDOR "Skycoder42") 
     set(CPACK_PACKAGE_CONTACT "Shatur <genaloner@gmail.com>") 
```

I wouldn't set `CPACK_DEBIAN_PACKAGE_MAINTAINER` because, according to the Debian Wiki:
> Debian Maintainers (DMs) are people who have a restricted ability to upload packages to the Debian archive.

Nevertheless, whomever is specified at `CPACK_PACKAGE_CONTACT` shows as the maintainer in Synaptic Package Manager.
<img width="625" alt="Screen shot of package properties in Synaptic Package Manager, showing Shatur <genaloner@gmail.com> as maintainer." src="https://user-images.githubusercontent.com/1840407/177775563-a421436d-9139-480a-b758-14c764d9ed9f.png">

Reply:
Your e-mail was not included in the merge. Should we revert and re-merge or do we leave it as is?